project.clj

```diff
@@ -73,7 +73,8 @@
             [lein-npm "0.6.2"]
             [lein-pdo "0.1.1"]]
 
-  :npm {:dependencies [["@sentry/node" "4.2.1"]
+  :npm {:dependencies [["@ensdomains/ens" "0.1.2"]
+                       ["@sentry/node" "4.2.1"]
```

Comment:
Why do you need sentry?

Reply:
It's a dependency that was there before. And is needed for service monitoring I guess.

Reply:
Oh, yeah I am blind. I thought that you added it just now by accident.